Source/Scene/Scene.js

```diff
@@ -1124,9 +1124,20 @@ define([
         if (command._dirty) {
             command._dirty = false;
             
+            var context = scene._context;
+            var derivedCommands = command.derivedCommands;
+
+            if (defined(scene.shadowMap)) {
```

Comment:
I'm going to leave this here for now so we don't have to track if `scene.shadowMap.enabled` changes to trigger creating the shadow derived commands.